addons/project/static/src/js/project_form.js

```diff
@@ -15,7 +15,7 @@ const ProjectFormController = FormController.extend({
         }
     },
     _onDomUpdated() {
-        const $editable = this.$el.find('.note-editable');
+        const $editable = this.$el.find('[name="description"]').children('.note-editable');
```

Comment:
This should be better:
```suggestion
        const $editable = this.$el.find('[name="description"] .note-editable');
```
🙂

Reply:
Yes, it is perfect. Changes done @xavierbol . Thank you for the changes. :slightly_smiling_face: